free-1.md

```diff
@@ -1,50 +1,61 @@
-# Free project 1
+# Free project 1: Music and Visual Art generated from keystrokes or text
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+This project could be used by anyone looking to create animations and musical notes from within their web browser. 
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+There isn't a particular need that the project would fulfill. Perhaps an innate "need to create art in a spontaneous and fun way and be able to share it with others". While a program like ContextFree is very methodical and has quite a steep learning curve, this project would allow users to be creative in a much easier and more engaging way while retaining the element of randomness that makes ContextFree interesting. 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+N/A
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I've always been fascinated by music visualization. My project draws inspiration from this website I came across on my Facebook newsfeed sometime last year: http://patatap.com/. Ever since I've seen it, I've wanted to build something like it, adding my own twists of course. 
 
 ### Domain
 _Describe the project's domain in five words._
 
+User-generated music and animations 
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+If I were to make an app that plays music in real time, the user could "interact with the langauge" through keystrokes. An alternative on a website could be through point-and-click but this may stifle creativity as a keyboard offers so many different options and would generally be more fun to play with. If it were a mobile app, touch could also be an option.
+
+Another option could be to have the music be generated from text. Users could type words, sentences or entire paragraphs which would be parsed, interpreted then played as notes (length of the word could relate to how long the note is, and the first letter of the word could relate to the pitch of the note).  
 
```

Comment:
This seems like a really cool idea! One of the interesting design spaces that comes to mind in this case is whether you want it to be easy / difficult for the user to create music that sounds really bad.  It doesn't seem like there's an obvious choice (though perhaps it's obvious to you).

On the one hand, having anything create good sounding and looking art is nice, since a user immediately feels productive, there is no frustration involved when their creations are of low quality.

On the other hand, it might feel more realistic if truly exceptional art is more difficult to create, as this could make the *true* art produced seem even more valuable and rewarding.

The reason I ask is because a mapping from paragraphs to sound in the manner you describe seems unlikely to produce sound which is enjoyable to listen to (as it would be similar to a seemingly random collection of pitches at various lengths). But perhaps this is what you intend, so that a program that both reads and sounds interesting would be much rarer and thus of greater significance?